utilities/compare_version.py

```diff
@@ -14,29 +14,34 @@
 
 
 def main():
-    pypi = json.load(urlopen("https://pypi.org/pypi/atc-dataplatform/json"))
-    pypi_version = parse(pypi["info"]["version"])
-    test_pypi = json.load(urlopen("https://test.pypi.org/pypi/atc-dataplatform/json"))
-    test_pypi_version = parse(test_pypi["info"]["version"])
+    pypi = get_versions("https://pypi.org/simple/", "atc-dataplatform")
+    pypi_test = get_versions("https://test.pypi.org/simple/", "atc-dataplatform")
 
     version = parse(atc.__version__)
 
-    if not (version > test_pypi_version):
-        print(
-            f"Current version {atc.__version__}"
-            f" is not ahead of TestPyPi's {test_pypi['info']['version']}"
-        )
+    if version in pypi_test:
+        print(f"Current version {atc.__version__} already exists in TestPyPi")
```

Comment:
use the variable "version" which is declared in line 20 ?

Reply:
so use "version" variable in stead of atc.__version__

Reply:
The version variable is a parsed version object which can have a string representation that is different from the actual version.

so as an example `parse('0.1.18pre1') == '0.1.18rc1'`

I therefore do not want to use the version variable.